### ISSUE TITLE
branch: molarityDescriberRefactor

### ISSUE THREAD

**Issue by zepumph** [phetsims/molarity#80]
To assist in #79.

**Comment by pixelzoom** [comment]
Reopening, this branch still exists.  @twant did you intend to delete the branch?

**Comment by twant** [comment]
@pixelzoom thanks for checking on this. Yes, apologies, I did mean to both close the issue and delete the branch. Will check with @zepumph to make sure the refactor looks good and then close & delete the branch in our meeting this coming Monday.

**Comment by twant** [comment]
deleting the branch and closing

**Comment by zepumph** [comment]
Should be re-deleted as part of phetsims/special-ops#198

**Comment by zepumph** [comment]
Deleted
